customTests/azure_gpu_amd_xgmi.nhc

```diff
@@ -16,7 +16,7 @@ check_gpu_xgmi(){
         error=${xgmi_errors[$i]}
         log "GPU $i has $error xgmi error(s)" 
         if [ $error -gt 0 ]; then
-            die 1 "check_gpu_xgmi: XGMI error count: $error" on GPU $i
+            die 1 "check_gpu_xgmi: XGMI error count: $error on GPU $i. FaultCode: NHCNA"
```

Comment:
This is an actual failure for AMD GPUs. AMD SKUs are not supported yet. Can we just revert the changes in this file and leave as is.